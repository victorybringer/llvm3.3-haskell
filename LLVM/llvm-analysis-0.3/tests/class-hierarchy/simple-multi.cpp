class Base1 {
public:
  Base1() {}
  virtual void f1() {}
};

class Base2 {
public:
  Base2() {}
  virtual void f2() {}
};

class Derived : public Base1, public Base2 {
  int m_i;
public:
  Derived(int);
  virtual int f3() const;
};

Derived::Derived(int i): m_i(i) {}
int Derived::f3() const {
  return m_i;
}
