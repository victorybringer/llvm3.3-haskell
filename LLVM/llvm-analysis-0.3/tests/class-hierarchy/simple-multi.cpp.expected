fromList [ ("Base1", fromList ["Derived"])
         , ("Base2", fromList ["Derived"])
         ]
